packages/core/src/navigation/navigation-start.element.ts

```diff
@@ -120,36 +123,70 @@ export class CdsNavigationStart extends LitElement implements FocusableItem {
 
   private handleStartButtonText() {
     spanWrapper(this.childNodes);
+
     // get the projected text now wrapped in a span and add the sr attribute.
     this.querySelector('span')?.setAttribute(NAVIGATION_TEXT_WRAPPER, '');
   }
 
+  private renderRootStart() {
+    return html`
+      <slot @slotchange=${this.handleStartButtonText}></slot>
+      <button
+        aria-pressed="${this.expandedRoot}"
+        aria-label="${this.expandedRoot ? this.i18n.navigationUnabridgedText : this.i18n.navigationAbridgedText}"
+        class="private-host"
+        id="${this.isGroupStart ? this.navigationGroupId : ''}"
+        type="button"
+      >
+        <div cds-layout="horizontal align:vertical-center">
+          <span class="icon-slot" cds-layout="${this.expandedRoot ? 'align:right' : 'align:left'}">
+            <slot name="cds-icon-slot">
+              ${this.startIcon
+                ? ''
+                : html` <cds-icon
+                    size="${!this.expandedRoot && this.isGroupStart ? 'xs' : 'sm'}"
+                    shape="${this.isGroupStart ? 'angle' : 'angle-double'}"
```

Comment:
Not a suggestion, just a note. To customize the icon here, end users would need to override either `angle` or `angle-double` in `ClarityIcons`...

Reply:
There is an `@querySlot` that checks for a projected icon with the [[cds-navigation-start-icon]](https://github.com/vmware/clarity/pull/6210/files/432432d88001480949acd08993505f6f692b8461#diff-e8fd8c90d14cc0e03558a42b2c491caa15b9028a3ca76685b64a96009faa3b59R102) attribute. If they put that on anything (hopefully an icon) we project that instead of the default icons. They catch is that if they customize it, then they need to manage its state if they want it to change or rotate etc. I worked up an early demo that used eye and eye-hide for the root navigation that controls the wide/narrow state of a navigation. I think it got removed when I re-worked the demos.